ledger/sandbox/src/main/scala/com/digitalasset/platform/index/JdbcIndex.scala

```diff
@@ -21,8 +22,12 @@ object JdbcIndex {
       ledgerId: LedgerId,
       participantId: ParticipantId,
       jdbcUrl: String,
-      metrics: MetricRegistry,
-  )(implicit mat: Materializer, logCtx: LoggingContext): Resource[IndexService] =
+      metrics: MetricRegistry
+  )(
+      implicit mat: Materializer,
+      logCtx: LoggingContext,
+      indexingOptions: IndexingOptions = IndexingOptions.defaultNoImplicitPartyAllocation)
```

Comment:
Not convinced about this being `implicit` and having a default value. Why not just make it a regular parameter? Implicits have this wonderful tendency to hang around long after we stop using them.

Reply:
Agreed, I'm going to revert to the explicit version and this will also solve your comments above.